apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -153,8 +154,9 @@ export class FilesStorageUC {
 
```

Comment:
In downloadBySecurityToken kann noch getSchoolId verwendet werden

Reply:
done